src/foam/nanos/controller/AppStyles.js

```diff
@@ -255,5 +262,31 @@ foam.CLASS({
       scroll-behavior: auto !important;
     }
   }
+
+
+  .large-subtitle {
+    font-size: 2.4rem;
+    line-height: 1.67;
+  }
+
+  .large-title {
+    text-align: center;
+    font-weight: bold;
+  }
+  @media only screen and (max-width: 320px) {
+    :root {
+      font-size: 50%;
```

Comment:
Wouldn't this change what 1rem is?

Reply:
yeah kind of, but for really small screens like iphone4 the design doesn't really fit the width without breaking the lines ... probably not the best solution tho, any suggestions?